tests/OSPSuite.HelpersForTests/DomainHelperForSpecs.cs

```diff
@@ -41,6 +42,22 @@ public static IDimension ConcentrationDimensionForSpecs()
          return _concentrationDimension;
       }
 
+      public static IEnumerable<IDimension> ExtendedDimensionsForSpecs()
+      {
+         var dimensions = new List<IDimension>();
```

Comment:
Why aren't you using the dimensions already defined in the file (concentration, fraction etc..). 
The NO_Dimension already exists also as a dImension normally so you code could be something like
reutnr new []{COnentrationDimForSPecs, NO_DIMENSION, FractionDimForSpecs).